docs/source/guide/rem-1-intro.myst

```diff
@@ -36,7 +34,7 @@ from mitiq.observable.observable import Observable
 from mitiq.observable.pauli import PauliString
 
 qreg = [LineQubit(i) for i in range(2)]
-circuit = Circuit(X.on_each(*qreg), measure_each(*qreg))
```

Comment:
How come you removed the measurement? Is it it not necessary when defining observables?

Reply:
Because when using an executor with to evaluate the expectation value of a Mitiq Observable, Mitiq automatically implements change of basis rotations and **adds measurements** on the qubits associated to the Observable. So we don't need to add measurements otherwise qubits will get measured twice.

The situation is different if instead of evaluating an Observable we want to just get the raw measurement results. In that case, the user must had measurements to the circuit.

I know, this is quite confusing and hopefully we'll make it simpler, but that's the current situation when using  `Observable`s in Mitiq.